README.md

```diff
@@ -31,6 +31,62 @@ to enter any S3 credential which quite rightly, developers do not really want to
 Note that this uses a server called 'minio' which is available for just about any platform.
 Minio has a web interface at http://s3.et.127.0.0.1.nip.io:3100 (unless you have changed the domain or port - then adjust as necessary)
 
+## External Dependencies
+
+### Docker Compose
+
+Minimum version: 1.21.0
+
+### Docker
+
+Minimum version:  18.05.0-ce
+
+## Quick Start
+
+If you just want to get going, here is a quick way to prove that it all works.
+
+```
+
+git clone --recursive git@github.com:ministryofjustice/et-full-system.git
+
+```
+
+If you want a different branch than develop (the default) then do this :-
+
+```
+git checkout <your_branch>
+git submodule update
+
+```
+
+then, irrespective of branch :-
+
+```
+
+cd et-full-system
+./bin/dev/test_server up
+
+```
+
+and wait for a message like this from docker
```

Comment:
Perhaps a really small string change to say something like

`and wait for a message like this, after three ruby gemlists are shown, from docker`

so that people know roughly how long it takes? They might be looking for this message from the start but it takes a while. Not that important but perhaps worth a mention.